autoinstall/FreeBSD/installerconfig

```diff
@@ -0,0 +1,129 @@
+
+# Set Proxy. without the proxy, maybe it will take more time to download pakages xorg & gnome
+{% if http_proxy_vm is defined and http_proxy_vm %}
+setenv HTTP_PROXY {{ http_proxy_vm }}
+{% endif %}
+
+# Installing packages
+echo "Installing packages ..." > /dev/ttyu0
+env ASSUME_ALWAYS_YES=YES pkg bootstrap -y
+env ASSUME_ALWAYS_YES=YES pkg update -f
+
+# Hit issue: reset by peer during install packages
+# The open-vm-tools is not installed by default
+mkdir -p /usr/local/etc/pkg/repos
+mount > /dev/ttyu0
+cp -rf /dist/packages/repos/FreeBSD_install_cdrom.conf /usr/local/etc/pkg/repos/FreeBSD_install_cdrom.conf
+env ASSUME_ALWAYS_YES=YES pkg update -f > /dev/ttyu0
+packages_to_install='bash sudo xorg kde5'
+for package_to_install in $packages_to_install
+do
+    ret=1
+    try_count=1
+    until [ $ret -eq 0 ]
+    do
+        echo "Install package $package_to_install (try $try_count time) ..." > /dev/ttyu0
+        env ASSUME_ALWAYS_YES=YES pkg install -y $package_to_install
+        ret=$?
+        try_count=$((try_count+1))
+    done
+    echo "The package($package_to_install) is already installed" > /dev/ttyu0
+done
+
+# Disable ISO repo and enable default repo
+rm -rf /usr/local/etc/pkg/repos/FreeBSD_install_cdrom.conf
+env ASSUME_ALWAYS_YES=YES pkg update -f > /dev/ttyu0
+packages_to_install='sddm open-vm-tools xf86-video-vmware xf86-input-vmmouse wget curl'
+for package_to_install in $packages_to_install
+do
+    ret=1
+    try_count=1
+    until [ $ret -eq 0 ]
+    do
+        echo "Install package $package_to_install (try $try_count time) ..." > /dev/ttyu0
+        env ASSUME_ALWAYS_YES=YES pkg install -y $package_to_install
+        ret=$?
+        try_count=$((try_count+1))
+    done
+    echo "The package($package_to_install) is already installed" > /dev/ttyu0
+done
+
+# Add new user. 
+{% if new_user is defined and new_user != 'root' %}
+echo "{{ vm_password }}" | pw useradd {{ new_user }} -s /bin/sh -d /home/{{ new_user }} -m -g wheel -h 0
+echo '{{ new_user }} ALL=(ALL:ALL) ALL' >> /usr/local/etc/sudoers
+{% endif %}
+
+# Set password of root user
+echo "{{ vm_password }}" | pw -V /etc usermod root -h 0
+
+# Enable root login via ssh
+echo "Enable root login via ssh ..." > /dev/ttyu0
+mkdir -p -m 700 /root/.ssh
+echo "{{ ssh_public_key }}" > /root/.ssh/authorized_keys
+chown -R root /root/.ssh
+chmod 0644 /root/.ssh/authorized_keys
+# We can't ssh to VM with empty password for root user
+sed -i .bak -e 's/#PasswordAuthentication.*/PasswordAuthentication yes/' /etc/ssh/sshd_config
+sed -i '' -e 's/#PermitRootLogin.*/PermitRootLogin yes/' /etc/ssh/sshd_config
+
+# Enable service
+echo "Enable service ..." > /dev/ttyu0
+sysrc sshd_enable="YES"
+sysrc ntpd_enable="YES"
+sysrc ntpd_sync_on_start="YES"
+
+# Config KDE desktop
+echo "proc      /proc       procfs  rw  0   0" >> /etc/fstab
+echo "exec ck-launch-session startplasma-x11" > /root/.xinitrc
+echo "exec ck-launch-session startplasma-x11" > /home/{{ new_user }}/.xinitrc
```

Comment:
These configures are used when there is no sddm. Since you installed sddm in previous step, it should have some sddm configure to start the KDE at boot.

Reply:
Hit issue "A black screen and a cross-shaped cursor" when i input account and login in. 
So I remove the configuration items about logining to desktop and leave the configuration to run command "startx"

Even if we can't login to the KDE desktop at boot time automatically. 
User can easily launch the desktop environment by running one command 'startx 'after auto-install.

Best regards
Yuhua Zou

Reply:
With 13.2 RC2 32bit image, it succeed to login to desktop env at boot time. But not for 64bit image.
Difference:
With 13.2 RC2 32bit image, install kde5 from offical repo. 
With 13.2 RC2 32bit image, install kde5 from ISO.

Reply:
It works well now to login to the KDE desktop at boot time